### PR TITLE
fix: Remove instances of process variable from build output

### PR DESCRIPTION
**Pull request by imprashast**
This will basically fix the issue with process variable ending up on the browser bundle.

Should icons be released as stable before merging?
or
Should we test this with the team that reported this first before doing all stable release?

### CODE REVIEW

File: components/util/i18n.ts [hunk 1]
```diff
@@ -20,7 +20,7 @@ function detectLocale(): SupportedLocale {
      * Server locale detection. This requires e.g LANG environment variable to be set on the server.
      */
     const serverLocale =
-      process.env.NMP_LANGUAGE ||
+      process?.env?.NMP_LANGUAGE ||
```
[the remark this answers is not in the document]
Comment (imprashast) [reply components/util/i18n.ts:23]: No because those are 2 different variables :)

[the remark this answers is not in the document]
Comment (BalbinaK) [reply components/util/i18n.ts:23]: 🙈 Of course, those are two different things. The `process.env.NODE_ENV` was supposed to replace a `process.env.NODE_ENV !== "production"` check in `@lingui/core`.

**Review by BalbinaK — APPROVED** [review]
> Should icons be released as stable before merging?

If that's required to properly test whether this change fully works on Eik, then I'd say it makes sense to make a stable release of @warp-ds/icons and update it here.  

> Should we test this with the team that reported this first before doing all stable release?

I'd say it would help us see right away if this actually fixes the previous issues. Can they test using the `-next.x` version of Vue from Eik?

**Comment by imprashast** [comment]
Yeah lets ask them to test with the next version first 👍

**Pull request merged**